hdx_hapi/endpoints/get_population.py

```diff
@@ -74,7 +74,9 @@ async def get_populations(
     output_format: OutputFormat = OutputFormat.JSON,
 ):
     """
-    Return the list of populations
+    Population data from the UNOCHA Common Operational Datasets (COD) originally sourced UNFPA (UN Population Fund)
```

Comment:
Please change to "Baseline population data sourced and maintained by UNFPA (UN Population Fund).

Reply:
Fixed this